dev/release/verify-release-candidate.sh

```diff
@@ -282,6 +282,40 @@ ${ARROW_CMAKE_OPTIONS}
   popd
 }
 
+test_csharp() {
+  pushd csharp
+
+  if ! which dotnet > /dev/null 2>&1; then
+    if [ "$(uname)" == "Darwin" ]; then
+      DOTNET_URL=https://download.visualstudio.microsoft.com/download/pr/328d95ad-eb94-4d1f-a4a9-2a35dbfea719/4913918a65ae135af1c8823f04061708/dotnet-sdk-2.2.300-osx-x64.tar.gz
+    else
+      DOTNET_URL=https://download.visualstudio.microsoft.com/download/pr/5e92f45b-384e-41b9-bf8d-c949684e20a1/67a98aa2a4e441245d6afe194bd79b9b/dotnet-sdk-2.2.300-linux-x64.tar.gz
+    fi
+
+    curl -o dotnet.tar.gz $DOTNET_URL
+    mkdir $PWD/bin
+    tar xf dotnet.tar.gz -C $PWD/bin
+    PATH=$PWD/bin:$PATH
+  fi
+
+  dotnet test
+  mv dummy.git ../.git
+  dotnet pack -c Release
+  mv ../.git dummy.git
+
+  if ! which sourcelink > /dev/null 2>&1; then
+    dotnet tool install --tool-path $PWD/bin sourcelink
+    export DOTNET_ROOT=$PWD/bin
```

Comment:
Because the required library libhostfxr.so could not be found.

```sh
A fatal error occurred, the required library libhostfxr.dylib could not be found.
If this is a self-contained application, that library should exist in [/Users/shiro/dev/arrow/apache-arrow-0.14.0/csharp/bin/.store/sourcelink/3.1.1/sourcelink/3.1.1/tools/netcoreapp2.1/any/].
If this is a framework-dependent application, install the runtime in the default location [/usr/local/share/dotnet] or use the DOTNET_ROOT environment variable to specify the runtime location.
```

Reply:
Thanks for the tip.